### PR TITLE
[MFTF] User controls access to Adobe Stock images from Admin Panel in ACL Automation

### PR DESCRIPTION
**Pull request by engcom-Golf**
### Description (*)
Implemented MFTF test to cover scenario: "User controls access to Adobe Stock images from Admin Panel in ACL"

### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#198: [MFTF]: User controls access to Adobe Stock images from Admin Panel in ACL

### Manual testing scenarios (*)
Run functional tests

### CODE REVIEW

**Comment by diazwatson** [comment]
@Nazar65 I get this exception when running this test, could you please take a look?

```
ASI-198$ bin/mftf run:test UserControlsAccessToAdobeStockImagesFromAdminPanelInACLTest

DEPRECATION: Test StorefrontOnePageCheckoutJsValidationTest is missing required annotations.{"testName":"StorefrontOnePageCheckoutJsValidationTest","missingAnnotations":"stories"}

In ActionMergeUtil.php line 286:

  Section cannot use names more than once.
  	name: firstImageAfterSearch is used more than once. (Parent: AdobeStockSection)
  	in file: /adobestock/ext/magento/stock-integration/AdobeStockImageAdminUi/Test/Mftf/Section/AdobeStockSection.xml.
  Exception occurred parsing action at StepKey "disableWYSYWYG"
```

**Comment by engcom-Golf** [comment]
Hi @diazwatson ok

File: AdobeStockImageAdminUi/Test/Mftf/Test/UserControlsAccessToAdobeStockImagesFromAdminPanelInACLTest.xml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            <waitForPageLoad stepKey="waitForRolesGridLoad" />
+            <actionGroup ref="AdminDeleteRoleActionGroup" stepKey="deleteUserRole">
+                <argument name="role" value="adminRole"/>
+            </actionGroup>
+            <amOnPage url="{{AdminUsersPage.url}}" stepKey="goToAllUsersPage"/>
+            <waitForPageLoad stepKey="waitForUsersGridLoad" />
+            <actionGroup ref="AdminDeleteNewUserActionGroup" stepKey="deleteUser">
+                <argument name="userName" value="{{admin2.username}}"/>
+            </actionGroup>
+            <amOnPage url="admin/admin/auth/logout/" stepKey="amOnLogoutPage"/>
```
Comment (sivaschenko) [inline AdobeStockImageAdminUi/Test/Mftf/Test/UserControlsAccessToAdobeStockImagesFromAdminPanelInACLTest.xml]: please use logout action group

File: AdobeStockImageAdminUi/Test/Mftf/Test/UserControlsAccessToAdobeStockImagesFromAdminPanelInACLTest.xml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            <argument name="User" value="admin2"/>
+        </actionGroup>
+        <!--Log out of admin and login with newly created user-->
+        <actionGroup ref="logout" stepKey="logoutOfAdmin"/>
+        <actionGroup ref="LoginAsAdmin" stepKey="loginAsNewUser">
+            <argument name="adminUser" value="admin2"/>
+        </actionGroup>
+        <!-- verify that user can access to adobe stock image -->
+        <actionGroup ref="AdminAdobeStockOpenSearchModalActionGroup" stepKey="openAdobeStockPopup"/>
+        <grabAttributeFrom selector="{{AdobeStockSection.firstImageAfterSearch}}" userInput="src"
```
Comment (sivaschenko) [inline AdobeStockImageAdminUi/Test/Mftf/Test/UserControlsAccessToAdobeStockImagesFromAdminPanelInACLTest.xml:70]: Please use verification approach from `AdminUserAccessesAdobeStockPanelTest`

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the PR @engcom-Golf ! Can you please update the branch, adjust the test to the updates and add verification of requirements from the story https://github.com/magento/adobe-stock-integration/issues/42 : effect of corresponding resources turned on/off for a role.

File: AdobeStockImageAdminUi/Test/Mftf/Test/UserControlsAccessToAdobeStockImagesFromAdminPanelInACLTest.xml (hunk 1 above)
Comment (engcom-Golf) [reply AdobeStockImageAdminUi/Test/Mftf/Test/UserControlsAccessToAdobeStockImagesFromAdminPanelInACLTest.xml]: ok

File: AdobeStockImageAdminUi/Test/Mftf/Test/UserControlsAccessToAdobeStockImagesFromAdminPanelInACLTest.xml (hunk 2 above)
Comment (engcom-Golf) [reply AdobeStockImageAdminUi/Test/Mftf/Test/UserControlsAccessToAdobeStockImagesFromAdminPanelInACLTest.xml:70]: ok

**Comment by engcom-Golf** [comment]
Hi @sivaschenko thank you for review, i'm updated PR  due to new review comment, can you please review new changes ?

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**